src/setup/telemetry/debugTransport.ts

```diff
@@ -0,0 +1,112 @@
+import type {BaseTransportOptions, Transport, TransportRequest, TransportRequestExecutor} from '@sentry/core';
+import {createTransport} from '@sentry/core';
+
+/**
+ * Enable this to log Sentry telemetry data to console instead of sending to Sentry servers.
+ */
+const DEBUG_SENTRY_ENABLED = true;
```

Comment:
Do we want to have sentry logs enabled by default? Will it provide any value for regular developers who don't add new spans?